### PR TITLE
Strip phone numbers and remove dashes and spaces before validations.

### PR DESCRIPTION
**Pull request by manfredt27**
Also annotate models

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -1,8 +1,22 @@
...
+#
+# It's strongly recommended that you check this file into your version control system.
+
 ActiveRecord::Schema.define(version: 20141202172600) do
 
+  # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
-  create_table "active_admin_comments", force: true do |t|
+  create_table "active_admin_comments", force: :cascade do |t|
```
Comment (manfredt27) [inline db/schema.rb]: Not sure why these changes happened

Comment (BrayanZ) [inline db/schema.rb]: New rails version uses `:cascade` instead of `true`

**Comment by BrayanZ** [comment]
Configure the annotations to be on the bottom on the file rather than in the top

File: app/models/lost_report.rb [hunk 2]
```diff
@@ -14,4 +32,14 @@ class LostReport < ActiveRecord::Base
   def phone_numbers
     (super || []).reject(&:blank?)
   end
+
+  private
+
+  def strip_phone_numbers
```
Comment (BrayanZ) [inline app/models/lost_report.rb]: I think this method could be rewrite as follows:
`phone_numbers.map { |phone_number| phone_number.strip.gsub('-', '')}`

Comment (manfredt27) [inline app/models/lost_report.rb]: @BrayanZ Done

File: app/models/lost_report.rb [hunk 3]
```diff
@@ -14,4 +16,26 @@ class LostReport < ActiveRecord::Base
...
+    self.phone_numbers = phone_numbers.map { |phone_number| phone_number.gsub('-', '').gsub(' ', '') }
```
Comment (BrayanZ) [inline app/models/lost_report.rb]: use `.delete(" -")` instead of the `#gsub`s

**Comment by jramscr** [comment]
Hey guys, why do you think about to change `should` syntax to `expect`?

File: app/models/lost_report.rb [hunk 4]
```diff
@@ -14,4 +16,26 @@ class LostReport < ActiveRecord::Base
   def phone_numbers
     (super || []).reject(&:blank?)
   end
+
+  private
+
+  def strip_phone_numbers
+    self.phone_numbers = phone_numbers.map { |phone_number| phone_number.delete(' -') }
```
Comment (alvarezloaiciga) [inline app/models/lost_report.rb]: should probably `delete` all non number chars.

```ruby
phone_numbers.map! { |phone_number| phone_number.delete('^0-9') }
```

Comment (manfredt27) [inline app/models/lost_report.rb]: @alvarezloaiciga wouldn't it be better to allow only numbers on the input field (on the html level)? Users shouldn't write non-numeric characters there in the first place

**Comment by manfredt27** [comment]
Ready for review

**Comment by alvarezloaiciga** [comment]
failing tests, you need to fix that before someone can merge this.

**Comment by manfredt27** [comment]
@alvarezloaiciga those cucumber tests will have to be changed with the new forms that @robquesada is working on. Should I still fix them?

**Comment by robquesada** [comment]
@jramscr it's ok to use `should` instead of `expect` while we're using shoulda-matchers gem. But feel free if you want to change that.
